src/PrestaShopBundle/Resources/views/Admin/Translations/list.html.twig

```diff
@@ -2,19 +2,52 @@
 
 {% block content %}
 <div class="search-translation">
-    <h2>{{ 'Search'|trans({}, 'Admin.Actions') }}</h2>
-    <div class="hide alert alert-info">{{ '0 result found'|trans({}, 'Admin.Notifications.Error') }}</div>
-    <label>{{ 'Translation key or value (2 characters at least):'|trans({}, 'Admin.International.Feature') }}</label>
-    <form>
-        <input pattern=".{2,}" type="search" id="jetsSearch">
-        <input class="btn btn-default btn-sm search-button" type="submit" value="{{ 'Search'|trans({}, 'Admin.Actions') }}">
-        <input class="btn btn-default btn-sm" type="reset" value="{{ 'Reset'|trans({}, 'Admin.Actions') }}">
-    </form>
-    <div id="jetsContent">
-        {{ getTranslationsForms(translationsTree, theme)|raw }}
+  <h2>{{ 'Search translations'|trans({}, 'Admin.Actions') }}</h2>
```

Comment:
Admin.International.Feature

Admin.Actions is for general actions that can be found on every pages. This is specific to translations.